.travis.yml

```diff
@@ -41,7 +41,7 @@ script:
   - bundle exec rake "${FOREMAN_PLUGIN_NAME}:rubocop"
   - cd ../${FOREMAN_PLUGIN_NAME}
   - npm run test;
-  - # npm run coveralls;
+  - npm run publish-coverage;
```

Comment:
Does this require a specific package in npm?

Reply:
this is a script in our package.json: https://github.com/theforeman/foreman_rh_cloud/blob/master/package.json#L11
that comes from `foreman-js` that uploads the test coverage to https://coveralls.io/github/theforeman/foreman_rh_cloud

it should add the coverage line on each PR we have, though I guess it will be added only after this will get merge